### PR TITLE
Added Meta field

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
This PR adds a Meta field to the Tag serializer. This field returns technical information about the tag, like the target or owner.

### CODE REVIEW

**Comment by mariajgrimaldi** [comment]
Hey @andrey-canon, this meta field is used to return technical information about the tag. Do you think we could add another field to it?

**Comment by andrey-canon** [comment]
@mariajgrimaldi  this is only to show that information in the API response?

**Comment by mariajgrimaldi** [comment]
> @mariajgrimaldi this is only to show that information in the API response?

yes!

**Comment by andrey-canon** [comment]
> > @mariajgrimaldi this is only to show that information in the API response?
> 
> yes! @andrey-canon is just a serializer field

however you already show that information, in other fields, don't you?

**Comment by mariajgrimaldi** [comment]
> > > @mariajgrimaldi this is only to show that information in the API response?
> > 
> > 
> > yes! @andrey-canon is just a serializer field
> 
> however you already show that information, in other fields, don't you?

After this PR the information will be displayed only inside that field @andrey-canon

File: eox_tagging/api/v1/serializers.py [hunk 1]
```diff
@@ -21,19 +21,30 @@
...
     class Meta:  # pylint: disable=old-style-class, useless-suppression
         """Meta class."""
         model = Tag
-        fields = ('key', 'tag_value', 'tag_type', 'access', 'activation_date', 'expiration_date',
+        fields = ('meta', 'key', 'tag_value', 'tag_type', 'access', 'activation_date', 'expiration_date',
                   'target_id', 'owner_id', 'owner_type', 'target_type', 'status')
 
+    def get_meta(self, instance):
+        """Getter of read-only field that returns technical information."""
+        technical_info = {
```
Comment (andrey-canon) [inline eox_tagging/api/v1/serializers.py:40]:
```suggestion
        return {
```

**Comment by andrey-canon** [comment]
> > > > @mariajgrimaldi this is only to show that information in the API response?
> > > 
> > > 
> > > yes! @andrey-canon is just a serializer field
> > 
> > 
> > however you already show that information, in other fields, don't you?
> 
> After this PR the information will be displayed only inside that field @andrey-canon

because of the attribute write_only ?

**Comment by mariajgrimaldi** [comment]
> > > > > @mariajgrimaldi this is only to show that information in the API response?
> > > > 
> > > > 
> > > > yes! @andrey-canon is just a serializer field
> > > 
> > > 
> > > however you already show that information, in other fields, don't you?
> > 
> > 
> > After this PR the information will be displayed only inside that field @andrey-canon
> 
> because of the attribute write_only ?

yes!

**Comment by mariajgrimaldi** [comment]
Which other fields can we add to meta? @felipemontoya

**Comment by felipemontoya** [comment]
@mariajgrimaldi here are my thoughts on this.

The current information on a tag is something like:
```
{
    "key": "1d35747d-bfeb-429a-8b1f-b758506f227c",
    "tag_value": "premium",
    "tag_type": "subscription_level",
    "access": "PRIVATE",
    "activation_date": null,
    "expiration_date": null,
    "target_id": "doug.weatherbee+beta",
    "owner_id": "lightbendsandbox.edunext.io",
    "owner_type": "Site",
    "target_type": "User",
    "status": "VALID"
}
```

and the information in the meta I think should be the info that is not related to the tag instance, but instead the tag definition or the metadata of the tag creation:

```
{
    "key": "1234-bfeb-429a-8b1f-1234",
    "tag_value": "the value",
    "tag_type": "tag_defined_name",
    "access": "PRIVATE",
    "activation_date": null,
    "expiration_date": null,
    "target_id": "username",
    "status": "VALID"
    "meta": {
        "owner_type": "Site",
        "target_type": "User",
        "owner_id": "customer.edunext.io",
        "created_at" ...
        "invalidated_at" ...
    }
}
```

**Pull request merged**